### DISCUSSION TITLE
Share of people who Aged 65+ that have fully vaccinated

### DISCUSSION THREAD

**Discussion by RuijingZ** [owid/covid-19-data#2761] [General]
Hi,

I am curious about the COVID-19 Explorer. I chose 'Aged 65+' in left column, and chose 'people fully vaccinated' and checked 'related to populations' at the top. Does that mean it refers to the share of people who aged 65+ that have fully vaccinated? Or if it does not matter which age group I choose, the graph is simply share of total amount of people that have fully vaccinated? Since I feel that the chart does not change when I change to 'Aged 70+' or 'median age'. Here is the site: https://ourworldindata.org/covid-vaccinations

![image](https://user-images.githubusercontent.com/68354122/204655222-c50b45a7-9c57-40d5-896c-b579778f62a8.png)

**Comment by lucasrodes** [comment]
Hi @RuijingZ,
Thanks for your message.

TL;DR: No, the line you observe measures the number of people fully of _all_ the population. I.e. it is a percentage relative to the population.


By selecting "Share aged 65+", you are just sorting the items in the left navigation menu so that countries with a higher share of people 65+ are shown first. That can be useful if you are interested in seeing a specific metric in countries where the population is a bit older.
